### PR TITLE
Remove from/import and update to PHP5.4.

### PR DESCRIPTION
**Pull request by Metalaka**

### CODE REVIEW

File: Cli.php [hunk 1]
```diff
@@ -182,10 +158,10 @@ protected function _addRule ( $visibility, $id, Array $methods, $pattern,
                     ? 'Method %s is'
                     : 'Methods %s are') .
                 ' invalid for the rule %s (valid methods are: %s).',
-                1, array(implode(', ', $diff), $id,
-                         implode(', ', self::$_methods)));
+                1, [implode(', ', $diff), $id,
+                    implode(', ', self::$_methods)]);
```
Comment (Hywan) [inline Cli.php]: Maybe we could join line 162 with 161 without having more than 80 columns?

File: Http.php [hunk 2]
```diff
@@ -250,14 +226,14 @@ protected function _addRule ( $visibility, $id, Array $methods, $pattern,
                     ? 'Method %s is'
                     : 'Methods %s are') .
                 ' invalid for the rule %s (valid methods are: %s).',
-                1, array(implode(', ', $diff), $id,
-                         implode(', ', self::$_methods)));
+                1, [implode(', ', $diff), $id,
+                    implode(', ', self::$_methods)]);
```
Comment (Hywan) [inline Http.php]: Same here.

**Comment by Hywan** [comment]
Good job!

File: Cli.php (hunk 1 above)
Comment (Metalaka) [inline Cli.php]: Fixed.
(81 columns)

**Comment by Hywan** [comment]
Thank you very much!

**Comment by Hywan** [comment]
http://wiki.hoa-project.net/projects:central:todo has been updated.

**Pull request merged**